### DISCUSSION TITLE
About IMF and stellar metallicity in CIGALE

### DISCUSSION THREAD

**Discussion by HoongWahSiew** [mboquien/cigale#41] [Q&A] [answered]
Hi, I am a CIGALE beginner and have some questions about the parameters in CIGALE,
1) How is the initial mass function (IMF) used in CIGALE? The distribution of masses of stars formed at every time instant is determined by IMF, isn’t it? 
2) Is constant stellar metallicity over time assumed in CIGALE? And that value is equal to the output “stellar metallicity” of BC03, isn’t it? I am considering using this metallicity to calculate the total mass of metal in a galaxy.

**Comment by Guang91** [comment, marked as answer]
1. Yes, the mass distribution is determined by IMF. Actually, the IMF is implemented by the single stellar population (SSP) library, i.e., Bruzual & Charlot (2003) or Maraston (2005), not directly by CIGALE.
2.  Yes, it's constant over time, and it's in the output stellar metallicity. Again, metallicity is implemented by the SSP library. So, you may want to read the paper of Bruzual & Charlot (2003) or Maraston (2005)

**Reply by HoongWahSiew** [reply]
Thank you very much for your reply! 🫡
